src/main/java/org/voltdb/meshmonitor/cli/MeshMonitorCommand.java

```diff
@@ -83,8 +83,8 @@ public class MeshMonitorCommand implements Callable<Integer> {
 
     @CommandLine.Option(
             names = {"-m", "--metrics-bind"},
-            description = "Bind address of a metrics server port in format ipv4[:port]",
-            defaultValue = "127.0.0.1:12223",
+            description = "Bind address for metrics server: [host][:port], e.g. 0.0.0.0:12223 for all IPv4 interfaces, [::]:12223 for all IPv6 interfaces, 12223 (default) for all interfaces (IPv4 and IPv6)",
```

Comment:
I don't believe [::] gets **only** IPv6.  Have you checked you cannot connect with IPv4?

Reply:
I've read that it depends on the OS and sysctl settings. On my mac, this code results in a dual stack binding:
```
ServerSocket s = new ServerSocket();
s.bind(new InetSocketAddress("::", 12223));
```
```
netstat
tcp46      0      0  *.12223                *.*                    LISTEN     
```
I will take out the "e.g. ..." so it just says what the default does, which is correct.